literate_notebooks/src-ES/05_columns.jl

```diff
@@ -0,0 +1,188 @@
+# # Introducción a DataFrames
+# **[Bogumił Kamiński](http://bogumilkaminski.pl/about/), May 23, 2018**
+# (Traducción por Miguel Raz Guzmán Macedo, Abril 17, 2021)
+
+using DataFrames 
+
+# ## Manipulando columnas de DataFrames
+
+#-
+
+# ### Renombrando columnas
+# 
+# Empecemos con un `DataFrame` de `Bool`s y nombres de columnas por default.
+
+x = DataFrame(Bool, 3, 4)
+
+# Con `rename`, creamos un nuevo `DataFrame`; aquí renombramos la columna `:x1` a `:A`. (`rename` sólo acepta colleciones de Pairs.)
+
+rename(x, :x1 => :A)
+
+# Con `rename!` hacemos una transformación in situ.
+# 
+# Esta vez hemos aplicado una función a cada nombre de columna.
+
+rename!(c -> Symbol(string(c)^2), x)
+
+# Podemos tambíen cambiar el nombre particular de una columna sin conocer el nombre original.
+# 
+# Aquí cambiamos el nombre de la tercera columna, creando un nuevo `DataFrame`.
+
+rename(x, names(x)[3] => :third)
+
+# Con `names!`., podemos cambiar el nombre de todas las variables.
+
+names!(x, [:a, :b, :c, :d])
+
+# Obtenemos un error si proveemos nombres duplicados
+
+names!(x, fill(:a, 4))
+
+#  A no ser que pasemos el argumento `makeunique=true`, lo cual nos permite pasar nombres duplicados.
+
+names!(x, fill(:a, 4), makeunique=true)
+
+# ### Reorganizar columnas
+
+#-
+
+# Podemos reorganizar el vector `names(x)` como sea necesario, creando un nuevo DataFrame en el proceso.
+
+srand(1234)
```

Comment:
Here is one example @bkamins, but they are everywhere...

Reply:
This should be `Random.seed!(1234)`. Where did you get `srand` from? Here is the source:
```
using Random
Random.seed!(1234)
x[:, shuffle(names(x))]
```

Reply:
We can easily replace with a regex in both portuguese and spanish. I just followed the `.jl` converted by @miguelraz

Reply:
As commented on Slack it seems that @miguelraz did not use `master` for doing the conversion. The fix should be easy hopefully.